src/components/Loans/LoansTabs.tsx

```diff
@@ -0,0 +1,87 @@
+import { Typography, Box, tabsClasses, Tabs, Tab } from '@mui/material';
+import React, { useEffect } from 'react';
+import LoansTabsList from './LoansTabsList';
+import TabPanel from './TabPanel';
+// eslint-disable-next-line import/extensions
+import data from '../../data/loans.json';
+
+interface TabItem {
+  approvedLoans: Loan[],
+  requests: Loan[],
+  activeLoans: Loan[],
+  closedLoans: Loan[],
+  rejectedLoans: Loan[],
+}
+
+const LoansTabs: React.FC = () => {
+  const { loanRequests }: any = data;
+
+  if (!loanRequests.length) {
+    return (
+      <p>No data found :(</p>
+    );
+  }
```

Comment:
Nice "Guard" 👍
Now we can prevent the case `loanRequests` is not array.
Here we can also prevent the case `loanRequests` is empty array. (`[]`)

```suggestion
  if (!loanRequests.length || loanRequest.length === 0) {
    return (
      <p>No data found :(</p>
    );
  }
```

Reply:
Yes, my bad. I didn't consider an empty array. Thanks for pointing it out :)